tests/rust/bitfield/cbindgen.toml

```diff
@@ -0,0 +1,3 @@
+[parse.expand]
```

Comment:
Instead of making this a nightly-only test via parse.expand you could instead use the `defines` section with something like `"feature = cbindgen" = "CBINDGEN"` and `#define CBINDGEN` in the `header` to effectively ifdef stuff out, but your call.

Reply:
I think I'll keep the existing version, translation of `cfg`s into `ifdef`s doesn't currently work on fields (https://github.com/eqrion/cbindgen/pull/316), and the version with expansion is closer to reality since you don't really want those `ifdef`s in the generated code in this case.